IoTuring/Entity/Deployments/Lock/Lock.py

```diff
@@ -13,10 +13,16 @@
         'base': 'pmset displaysleepnow'
     },
     OsD.LINUX: {
-        'gnome': 'gnome-screensaver-command -l',
+        # 'gnome': 'gnome-screensaver-command -l', as of my knowledge, gnome-screensaver is deprecated
```

Comment:
Yes, but it's still available in a lot of distros: https://pkgs.org/download/gnome-screensaver

Latest debian and Ubuntu LTS still have it.

Reply:
I'm ok with dropping support for gnome-screensaver, but than fully remove it, not just comment. We can get it back from git history anyway.

Reply:
I added it again, since i think dropping support for debian and Ubuntu LTS has such a big userbase.